### DISCUSSION TITLE
缺少libdc1394-22-dev包

### DISCUSSION THREAD

**Discussion by belirux** [linuxdeepin/developer-center#5444] [Troubleshooting | 故障排查] [answered]
我在编译安装opencv时 提示缺包
![image](https://github.com/linuxdeepin/developer-center/assets/138775053/a0e2bd89-c547-45d5-af36-61cef974f31a)

**Comment by BLumia** [comment]
什么环境，尝试构建的是什么版本，依赖列表是哪来的？

如果是重新编译系统的opencv的话，鉴于你没说你的环境是什么我就先假定你是 v23 beta，那 `apt build-dep libopencv-mobile` 就可以安装所有所需的构建依赖。

**Reply by belirux** [reply, marked as answer]
我是按照https://github.com/hybridgroup/gocv#ubuntulinux 文档编译安装的， 我修改Makefile文件去掉libdc1394-22-dev后编译成功
